contrib/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/envoyproxy/go-control-plane/contrib
 
-go 1.21
+go 1.22.8
```

Comment:
nit: it would be great if the go.mod version can remain the same for this fix.

Reply:
This version is automatically updated due to the added dependency on the root-level go.mod which has been using 1.22.8 for some time.
If a blocker I can see to rollback this change but it is not, per se, part of the module split